src/pages/tools.js

```diff
@@ -10,7 +10,7 @@ import { FormattedMessage } from 'react-intl'
 
 const Tools = ({ pageContext: { locale } }) => {
   return (
-    <Layout locale={locale}>
+    <Layout locale={locale} langSwitchable={true}>
```

Comment:
Is the tools page also needed?

Reply:
Yes, the tool page also need language switchable